modules/contentpool_custom_elements/contentpool_custom_elements.info.yml

```diff
@@ -0,0 +1,5 @@
+type: module
+name: Contentpool Custom elements
+description: 'Implements custom elements for Contentpool'
+package: Contentpool
+core: '8.x'
```

Comment:
please add configuration to the contentpool which renders the article paragraph field as custom-elements, so this is actually applied.
next step is using this during replication, but that's another issue then.

Reply:
i don't really know what you mean, what should be applied?